ScrollToTop FAB Added

dhwaj1902 opened the pull request:
Added Scroll To Top Button is a Floating Action Button, and it triggers when the page offset is greater than 400.
![image](https://user-images.githubusercontent.com/57860123/115498644-9510eb80-a28b-11eb-8229-41721e430fb1.png)

Advantages:
No need to add a separate ScrollToTop button on every page.

adarsh-technocrat commented:
Good change! I would suggest making a fab button offset above the bottom navbar.

dhwaj1902 commented:
@adarsh-technocrat done☺.

![image](https://user-images.githubusercontent.com/57860123/115500001-1bc6c800-a28e-11eb-8845-9aaabd1e8983.png)

Abhijay007 reviewed (commented):
@dhwaj1902 I reviewed your code and I want to suggest some minor changes :

1. If you are implementing this scroll-to-top fab then first remove predefined code for the scroll-to-top button from all pages and then implement this.

![inconsitent ui](https://user-images.githubusercontent.com/64387054/115514455-bb418600-a2a1-11eb-8e27-4255f94654d6.PNG)

2. And try to fit code within the UI as we are making this app for android and I know when we use inspect it stays in position but for the code, consistency try to fit it inside UI space ( ref above image )

3. Remove this back-to-top button from the profile page footer as I think that we don't need it on that page because if we resolve that page height issue then the footer will stick to the bottom. ( I am not sure about this you may need to confirm it from the mentor ).

![profile_footer](https://user-images.githubusercontent.com/64387054/115515875-2d669a80-a2a3-11eb-88bd-6ae784d73c49.PNG)

dhwaj1902 commented:
Hey, @Abhijay007 thanks for reviewing.

1. In your first suggestion, I found it amazing and have removed the previous scroll to top button from the following pages:
     - Home Page
     - Course Page
     - Wish List
     - Peer Discussion Page

2. While doing this, I found on Routes Page that the routes of ```terms-and-conditions```, ```about``` and ```faq``` are repeated as shown in the picture.
![image](https://user-images.githubusercontent.com/57860123/115532562-8c341000-a2b3-11eb-81ac-51288459fef0.png)
So, I have removed it.

3. As in your second suggestion, since we are making an app, I think so we are not focussing on that how it looks on the web. That's totally my view but if you want then I can implement it, but the problem is that I have added the following lines for making it fixed on the right bottom, so if I changed it then it will also impact mobile view.
```
right: "10px",
bottom: "65px",
```

4. If we removed the height issue, then I have done it in a way that if page offset is greater than 400 then it will show, else it will not. 

Sorry if I am not able to understand what you are trying to say.

dhwaj1902 commented:
@Abhishek-kumar09 Please look into this Pr.

Abhishek-kumar09 commented:
We are gonna remove the scroll to top button in the future version of app. Currently, the scroll to top is looking with center aligned and full color opacity.

Abhijay007 commented:
@dhwaj1902 you can close this PR as we may don't need back to top button in the future versions of the application.
